core-bundle/contao/themes/flexible/styles/main.css

```
@@ -686,7 +686,7 @@ h2.sub_headline {
 	width: 20%;
 }
 
-.tl_limit .tl_select {
+.tl_limit .tl_select_wrapper {
```

```suggestion

.tl_select_wrapper {
    margin: 3px 0;
}

.tl_select_wrapper > * {
    margin: 0;
}

.tl_limit .tl_select_wrapper {
```

1. `.tl_select` previously had a `margin: 3px 0` that was necessary for the label (e. g. check it in any `type` selection within a content element). This however won't take any effect within `.tl_filter` so that should be fine here.

2. We also did reset the margin bottom with `choices:last-child` but in our case since we are wrapping it entirely, I'd go for `.tl_select_wrapper > * { margin: 0 }` to also complete any unnecessary CLS on a Turbo disconnect when it becomes a `select` for a short time. Basically the `tl_select` class was handling everything before so we'd want at least the layout and spacings to be the same.

---

@zoglo is this the only change? Otherwise feel free to directly change stuff in this Pull Request if you found issues in styling, that would be a lot easier (and I fully trust you on that) 😊

---

See 5695869eb9b7afd744aa0e623b96455868fe93d5 and 5f07ab15a92559969b69f63a0cd921680f81c1f3